tests/gui/anaconda-autogui-testing.ks

```diff
@@ -10,7 +10,7 @@ rootpw qweqwe
 bootloader --location=mbr
 zerombr
 clearpart --all
-part / --fstype="ext4" --size=6000
+part / --fstype="ext4" --size=10240
```

Comment:
Have you been hitting problems? The tests are all minimal package sets so really 4G should be enough and 6G more than enough.